src/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -640,3 +640,83 @@ public static partial class ReflectionModelServices
         public static bool TryMakeGenericPartDefinition(System.ComponentModel.Composition.Primitives.ComposablePartDefinition partDefinition, System.Collections.Generic.IEnumerable<System.Type> genericParameters, out System.ComponentModel.Composition.Primitives.ComposablePartDefinition specialization) { throw null; }
     }
 }
+namespace System.ComponentModel.Composition.Registration
```

Comment:
Sorry for the late reply by why are these going into System.ComponentModel.Composition as opposed to System.ComponentModel.Composition.Registration? We cannot combine these and remain compatible with .NET Framework.